restaurant/src/assets/menu.css

```diff
@@ -0,0 +1,42 @@
+.menuheadpic {
+  background-image: url("menuteahead.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 500px;
+}
+.menu-page {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  justify-content: center;
+  align-items: center;
+  background-image: url("teabg.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 100%;
+}
+.menu-card {
+  display: flex;
+  flex-direction: column;
+  padding: 5% 25%;
```

Comment:
It is better to set padding in pixels, setting padding in % makes it change a lot depending on the width and height of the screen.